state/execution.go

```diff
@@ -220,7 +220,7 @@ func (blockExec *BlockExecutor) ApplyBlock(
 		return state, err
 	}
 
-	blockExec.logger.Info(
+	blockExec.logger.Debug(
```

Comment:
This one also was useful today (well, actually its `v0.34.x` counterpart: "executed block") to delimit the logs that were coming from the block execution itself. I think it should stay in Info